cyraft/election_watchdog.py

```diff
@@ -29,7 +31,9 @@ async def run(self) -> None:
         _logger.debug("Starting election watchdog")
         loop = asyncio.get_event_loop()
         self._next_election_timeout = loop.time() + self.election_timeout
-        self._election_timer = loop.call_at(self._next_election_timeout, self.on_election_timeout)
+        self._election_timer = loop.call_at(
+            self._next_election_timeout, self.on_election_timeout
+        )
```

Comment:
Ok, please use the following settings in VSCode:

```
{
    "editor.defaultFormatter": "ms-python.black-formatter",
    "black-formatter.args": [
        "--line-length=120"
    ],
    "editor.formatOnSave": true
}
```

I might have forgotten to mention we use a line length of 120.

https://wiki.zubax.com/en/public/development/Zubax-Python-coding-conventions

Reply:
[Fixed](https://github.com/OpenCyphal-Garage/cyraft/pull/11/commits/e9cd8165d7ba66c545d75d3037f5b044045cc01d)